### PR TITLE
docs: Add source links to RECOMMENDED.md

### PR DESCRIPTION
**Pull request by ghost**
The file didn't contain any sources to back up the claims made, so I added them.

### CODE REVIEW

File: RECOMMENDED.md [hunk 1]
```diff
@@ -1 +1 @@
-GNOME is the only desktop that secures privileged wayland protocols like screencopy. This means that on non-GNOME systems, applications can access screen content of the entire desktop. This implicitly includes the content of other applications. It's primarily for this reason that GNOME images are recommended.
\ No newline at end of file
+GNOME is currently the only desktop that secures privileged wayland protocols like screencopy. KDE and Sway both have attemps at fixing this, but do not have a working implementation. <sup>[KDE](https://invent.kde.org/plasma/xdg-desktop-portal-kde/-/issues/7)</sup> <sup>[Sway](https://github.com/swaywm/sway/issues/2333)</sup> This means that on non-GNOME systems, applications can access screen content of the entire desktop. This implicitly includes the content of other applications. It's primarily for this reason that GNOME images are recommended.
```
Comment (RoyalOughtness) [inline RECOMMENDED.md]: this is implied/redundant

Comment (RoyalOughtness) [inline RECOMMENDED.md]: typo: attempts

Comment (RoyalOughtness) [inline RECOMMENDED.md]: This is not really accurate. They have proposals/open issues, but not any fixes yet.

**Review by RoyalOughtness — CHANGES_REQUESTED** [review]

File: RECOMMENDED.md (hunk 1 above)
Comment (RoyalOughtness) [reply RECOMMENDED.md]: this was referring to the word "currently", github doesn't let me tag specific words

File: RECOMMENDED.md [hunk 2]
```diff
@@ -1 +1 @@
-GNOME is the only desktop that secures privileged wayland protocols like screencopy. This means that on non-GNOME systems, applications can access screen content of the entire desktop. This implicitly includes the content of other applications. It's primarily for this reason that GNOME images are recommended.
\ No newline at end of file
+GNOME is currently the only desktop that secures privileged wayland protocols like screencopy. Although KDE and Sway have plans to fix this. <sup>[KDE](https://invent.kde.org/plasma/xdg-desktop-portal-kde/-/issues/7)</sup> <sup>[Sway](https://github.com/swaywm/sway/issues/2333)</sup> This means that on non-GNOME systems, applications can access screen content of the entire desktop. This implicitly includes the content of other applications. It's primarily for this reason that GNOME images are recommended.
```
Comment (RoyalOughtness) [inline RECOMMENDED.md]: Probably don't need "although"

Comment (ghost) [reply RECOMMENDED.md]: fixed

File: RECOMMENDED.md (hunk 1 above)
Comment (ghost) [reply RECOMMENDED.md]: done

**Review by RoyalOughtness — APPROVED** [review]

**Review by RoyalOughtness — APPROVED** [review]

**Pull request merged**